src/resmom/mom_hook_func.c

```diff
@@ -1700,8 +1704,9 @@ run_hook(hook *phook, unsigned int event_type, mom_hook_input_t *hook_input,
 	}
 
 #ifdef WIN32
-	secure_file(file_out, "Administrators",
-		READS_MASK|WRITES_MASK|STANDARD_RIGHTS_REQUIRED);
+	if(secure_file(file_out, "Administrators",
+		READS_MASK|WRITES_MASK|STANDARD_RIGHTS_REQUIRED) == 0) 
+		log_err(-1, __func__, "Failed to change permissions of the hook output file");
 #endif
 
 
```

Comment:
line 1729 
	if (py_script != NULL) {
should we log error when py_script == NULL in the function?

Reply:
Done in https://github.com/PBSPro/pbspro/pull/1737/commits/06f24712f1b101a386d5b88656a24489250e52f0